utils/rpms/daos.spec

```diff
@@ -422,6 +423,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %attr(4750,root,daos_server) %{_bindir}/daos_firmware
 
 %changelog
+* Fri Apr 23 2021 Steven Bollinger <stevenx.bollinger@intel.com> 1.3.0-14
```

Comment:
You need to roll Release to 14 on line 11 of this file

Reply:
Fixed.